src/main/java/mate/academy/mybookshop/dto/UpdateCategoryRequestDto.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.mybookshop.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record UpdateCategoryRequestDto(@NotBlank String name, String description) {
```

Comment:
I do not see a big differance in create/update category request DTOs

Reply:
yeah, maybe you right. I just thought that it be nice for future changes.